pkgs/applications/misc/etesync-dav/default.nix

```diff
@@ -1,38 +1,23 @@
 { lib
 , stdenv
-, python3
+, python39
```

Comment:
Is there any reason to use explicit python version?

Reply:
Sadly, yes, older version don't work.